near-sdk-sim/README.md

```diff
@@ -1,16 +1,22 @@
-# Near Simulator
+# NEAR Simulator & cross-contract testing library
 
-This crate contains tools for simulating transactions on NEAR protocol.
-It also allows testing cross contract calls and inspecting the intermediate state of the transactions
-involved.
+When writing NEAR contracts, with Rust or other Wasm-compiled languages like [AssemblyScript](https://github.com/near/near-sdk-as), the default testing approach for your language of choice (such as `mod test` in your Rust project's `src/lib.rs` file) is great for testing the behavior of a single contract in isolation.
 
-# Documentation
-To view locally, clone this repo and from this folder run `cargo doc --open`.
+But the true power of blockchains & smart contracts comes from cross-contract calls. How do you make sure your cross-contract code works as you expect?
 
-## Setup
+As a first step, you can use this library! With it, you can:
 
-Currently this crate depends on a the github repo of [nearcore](https://github.com/near/nearcore), so this crate must be a git dependency too.
-Furthermore, this crate's dependencies conflict with building the Wasm smart contract, so you must add it under the following:
+* Test cross-contract calls
+* Accurately check [gas](https://docs.near.org/docs/concepts/gas) & [storage](https://docs.near.org/docs/concepts/storage-staking) usage for your contract
```

Comment:
This isn't completely true, but you can profile the gas usage to understand where your costs are going.  We found that the gas used on testnet was higher so this does provide a lower bound on gas usage.

Reply:
How about I change to:

```suggestion
* Profile [gas](https://docs.near.org/docs/concepts/gas) & [storage](https://docs.near.org/docs/concepts/storage-staking) usage for your contract, establishing lower bounds for costs of deployed contracts and rapidly identifying problematic areas prior to deploying.
```